Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY src ./src
 RUN cargo install --path .
 
 
-FROM debian:bullseye-20230502-slim
+FROM debian:bullseye-20240904-slim
```

Comment:
It could be really nice to actually pin the hash of the image. Not doing it broke the image without touching to the dockerfile due to lib `libssl3.so.3` no longer being present in libssl-dev but now now in `libssl3`

Reply:
Maybe something like `debian:12-slim@sha256:ad86386827b083b3d71139050b47ffb32bbd9559ea9b1345a739b14fec2d9ecf` (latest found hash)